app/messenger.py

```diff
@@ -0,0 +1,12 @@
+from app import messages_en, messages_pl
+from app.messages_enums import Message
+
+
+def send(message: Message, lang: str):
```

Comment:
Naming this function `send` can give the wrong idea that this functions actually sends something (e.g.: an email), when actually it only generates a message in a certain language. Maybe it could be called generate, or any other word (as long as it is short and not misleading)

Reply:
You're right, it's not the best name. `generate` seems ok, but what about `translate`?

Reply:
My personal opinion is that `translate` is about an input word being translated into an output word?! And what you are doing is like a mapping instead of transforming. you are mapping a key to a value which is a message in a certain language. So the input was not translated from language X to Y. What do you think about this?

Reply:
Makes sense, too.

...

Eureka! What about `mapMessage`?